config/config.hjson

```diff
@@ -9,4 +9,30 @@
     '''
     (fag(g|got|tard)?\b|cock\s?sucker(s|ing)?|ni((g{2,}|q)+|[gq]{2,})[e3r]+(s|z)?|mudslime?s?|kikes?|\bspi(c|k)s?\b|\bchinks?|gooks?|bitch(es|ing|y)?|whor(es?|ing)|\btr(a|@)nn?(y|ies?)|\b(b|re|r)tard(ed)?s?)
     '''
+  # rate limits for various user actions, by user ip
```

Comment:
Must have mistakenly gotten added, rate limits are in the DB now.

Reply:
Right, fixed.